src/utils/daos.c

```diff
@@ -971,38 +971,50 @@ static int
 cont_op_hdlr(struct cmd_args_s *ap)
 {
 	daos_cont_info_t	cont_info;
-	int			rc;
-	int			rc2;
+	int			rc = 0;
+	int			rc2 = 0;
 	enum cont_op		op;
 	const int		RC_PRINT_HELP = 2;
 
 	assert(ap != NULL);
 	op = ap->c_op;
 
-	/* All container operations require a pool handle, connect here.
-	 * Take specified pool UUID or look up through unified namespace.
+	/* cont_clone uses its own src and dst variables, and does
+	 * not use the regular pool and cont ones stored in the
+	 * ap struct. This is because for a clone it is necessary
+	 * to explicitly specify whether a container is a src or
+	 * dst
+	 */
+	if (op == CONT_CLONE) {
+		rc = cont_clone_hdlr(ap);
+		return rc;
+	}
+
+	/* All container operations require a pool handle, connect
+	 * here. Take specified pool UUID or look up through unified
+	 * namespace.
 	 */
 	if ((op != CONT_CREATE) && (ap->path != NULL)) {
 		struct duns_attr_t dattr = {0};
 		struct dfuse_il_reply il_reply = {0};
 
-		ARGS_VERIFY_PATH_NON_CREATE(ap, out, rc = RC_PRINT_HELP);
+		ARGS_VERIFY_PATH_NON_CREATE(ap, out,
+					    rc = RC_PRINT_HELP);
 
 		/* Resolve pool, container UUIDs from path if needed
-		 *
-		 * Firtly check for a unified namespace entry point, then if
-		 * that isn't detected then check for dfuse backing the
-		 * path, and print pool/container/oid for the path.
+		 * Firtly check for a unified namespace entry point,
+		 * then if that isn't detected then check for dfuse
+		 * backing the path, and print pool/container/oid
+		 * for the path.
 		 */
 		rc = duns_resolve_path(ap->path, &dattr);
 		if (rc) {
-
 			rc = call_dfuse_ioctl(ap->path, &il_reply);
 			if (rc != 0) {
-				fprintf(stderr, "could not resolve pool, "
-					"container by path: %d %s %s\n",
+				fprintf(stderr, "could not resolve "
+					"pool, container by "
+					"path: %d %s %s\n",
```

Comment:
@dsikich Minor format:
(Only if pushing again)
Since this was shifted and then unshifted, we could just revert these lines to exactly what they were originally to simplify the diffs, and remove the need for extra lines.